packages/wallet/src/W3mFrameProvider.ts

```diff
@@ -330,20 +360,42 @@ export class W3mFrameProvider {
   ) {
     const { payload } = event
     if (payload) {
-      this.setSessionToken(payload.token)
+      // Add When Implemented:  this.setSessionToken(payload.token)
```

Comment:
```suggestion
      // TODO: Add When Implemented:  this.setSessionToken(payload.token)
```

If this will be handled later (not in this PR), could we make it with the following convention so we could follow todo comments better way

Reply:
Will change to more explicit `Ilja TODO` as we have no-todo rule in eslint